src/main/java/org/jenkinsci/plugins/docker/workflow/client/DockerClient.java

```diff
@@ -104,7 +104,7 @@ public String run(@Nonnull EnvVars launchEnv, @Nonnull String image, @CheckForNu
         if (args != null) {
             argb.addTokenized(args);
         }
-        
+
```

Comment:
I've moved the trailing whitespace cleanup into a separate commit so you can accept/reject that as you see fit.